calculate RTT if ACKs are not sent consistently (#577)
quicwg/multipath: draft-ietf-quic-multipath.md

to compute smoothed_rtt and rttvar per
{{Section 5.3 of QUIC-RECOVERY}} regardless of the path through which PATH_ACK frames are
received. This approach will provide good results

> as long as acknowledgements are sent consistently over one paths.

Author: ```suggestion
as long as acknowledgements are sent consistently over one path.
```

can, however, remember the path over which the PATH_ACK that produced
the minimum RTT was received, and restart the minimum RTT computation
if that acknowledgement path changes or is abandoned.
If acknowledgements are not send consistently over one path, the
acknowledgement receiver can monitor over which path acknowledgement

> are received any ony use samples for acknowledgements received on the same

Author: ```suggestion
are received and only use samples for acknowledgements received on the same
```

path than the data was sent, if any, or simply use only samples from the

> acknowledgement path with the largest delay.

Reviewer: Please delete the text after "if any", up to "largest delay." It is hard to find a theoretical justification for using "largest delay".

Author: Okay, will remove.
